File: check_evidence_node.js
const fs = require('fs');
const source = fs.readFileSync('/home/ubuntu/thai-order-project-vault/n8n/CUSTOMER_CHAT_EVIDENCE_PREPARE.js', 'utf8');
fs.writeFileSync('/tmp/customer_evidence_wrapped.js', `(function(){\n${source}\n})();\n`);

File: check_evidence_node.py
from pathlib import Path
source=Path('/home/ubuntu/thai-order-project-vault/n8n/CUSTOMER_CHAT_EVIDENCE_PREPARE.js').read_text()
Path('/tmp/customer_chat_evidence_wrapped.js').write_text('(function(){\n'+source+'\n})();\n')
